Ark.Tools.NLog/NlogConfigurer.cs

```diff
@@ -274,7 +275,7 @@ public Configurer WithApplicationInsightsTarget(string instrumentationKey, bool
             public Configurer WithConsoleTarget(bool async = true)
             {
                 var consoleTarget = new ConsoleTarget();
-                consoleTarget.WriteBuffer = async;
+                consoleTarget.ForceWriteLine = async;

```

Comment:
Sorry about the confusion but `ForceWriteLine` is inverted compared to `WriteBuffer`. So you probably want to do this:
```csharp
consoleTarget.ForceWriteLine = !async;
```